app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<section class="login-signup-container">
+
+<%= render "devise/shared/back_to_home" %>
+
+<div class="main-cont-logsign">
+
+  <div class="heading-text">
+    <h1>Sign Up</h1>
+  </div>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+
+<div class="column main-login">
+      <%= f.email_field :name, placeholder: "Name*",autofocus: true, autocomplete: "name", class: "input-default" %>
```

Comment:
- [ Required ] I think this field should not be an `email_field` this prevents a user to register on your application using their `name`. Please refactor it to the code below so that users find it easy to create accounts using their names on your webpage.

```ruby
<%= f.text_field :name, placeholder: "Name*",autofocus: true, autocomplete: "name", class: "input-default" %>
```

![Screenshot from 2023-04-28 13-03-36](https://user-images.githubusercontent.com/16392046/235119049-a6c8c27c-cc3e-4536-9ad4-fb5fafb55398.png)

Reply:
Well resolved ✔️